### PR TITLE
Add missing geoname ids to continents

### PR DESCRIPTION
**Pull request by dominikkukacka**
Add all the missing geoname ids to the continents

### CODE REVIEW

File: data/102/191/577/102191577.geojson [hunk 1]
```diff
@@ -820,6 +820,7 @@
         85633057
     ],
     "wof:concordances":{
+        "gp:id":"6255150",
```
Comment (thisisaaronland) [inline data/102/191/577/102191577.geojson:823]: This should be `gn:id`

Comment (dominikkukacka) [reply data/102/191/577/102191577.geojson]: You are right about that. Fixed

File: data/102/191/573/102191573.geojson [hunk 2]
```diff
@@ -1010,6 +1010,7 @@
         85633813
     ],
     "wof:concordances":{
+        "gn:id":"6255146",
```
Comment (stepps00) [inline data/102/191/573/102191573.geojson:1013]: possible to adjust this, from string to int?

**Comment by stepps00** [comment]
Aside from one value type, LGTM. Can merge when that's corrected. Thanks @dominikkukacka

File: data/102/191/573/102191573.geojson (hunk 2 above)
Comment (dominikkukacka) [reply data/102/191/573/102191573.geojson]: fixed

**Comment by dominikkukacka** [comment]
Hey, what's the status here? :)

**Comment by stepps00** [comment]
Hey @dominikkukacka your commits were merged via https://github.com/whosonfirst-data/whosonfirst-data-admin-xy/pull/41. There were some merge conflicts in this pull request, so we moved your changes to a new branch to untangle them and validate the records before merging.

**Comment by dominikkukacka** [comment]
Super. Thanks!!

**Pull request merged**